security/pkg/pki/ca/selfsignedcarootcertrotator.go

```diff
@@ -86,25 +87,25 @@ func NewSelfSignedCARootCertRotator(config *SelfSignedCARootCertRotatorConfig,
 // Run refreshes root certs and updates config map accordingly.
 func (rotator *SelfSignedCARootCertRotator) Run(stopCh chan struct{}) {
 	if rotator.config.enableJitter {
-		rootCertRotatorLog.Infof("Jitter is enabled, wait %s before "+
+		rootCertRotatorLog.Debugf("Jitter is enabled, wait %s before "+
 			"starting root cert rotator.", rotator.backOffTime.String())
 		select {
 		case <-time.After(rotator.backOffTime):
-			rootCertRotatorLog.Infof("Jitter complete, start rotator.")
+			rootCertRotatorLog.Debugf("Jitter complete, start rotator.")
 		case <-stopCh:
-			rootCertRotatorLog.Info("Received stop signal, so stop the root cert rotator.")
+			rootCertRotatorLog.Debug("Received stop signal, so stop the root cert rotator.")
 			return
 		}
 	}
 	ticker := time.NewTicker(rotator.config.CheckInterval)
 	for {
 		select {
 		case <-ticker.C:
-			rootCertRotatorLog.Info("Check and rotate root cert.")
+			rootCertRotatorLog.Debug("Check and rotate root cert.")
```

Comment:
I think these logs should remain "info" given they are in a critical path. Was is the primary concern with them remaining at this log level?

Reply:
Redundant - there is another info log if an actual rotation happen, and if it doesn't - it's just spam.